### PR TITLE
Prefer seed from uri over load

### PR DESCRIPTION
**Pull request by jackwaudby**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2010.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-2010.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2010.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-2010.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)

**Comment by NataliaIvakina** [comment]
Why is this PR labeled `2025.01`? We have this functionality in 5.x if I'm not mistaken

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 1]
```diff
@@ -85,12 +85,17 @@ By appending `DUMP DATA` to the command `DROP DATABASE`, you can create a dump o
 DROP DATABASE movies DUMP DATA
 ----
 
-These dumps are equivalent to those produced by xref:backup-restore/offline-backup.adoc[`neo4j-admin database dump`] and can be similarly restored using the xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`] command.
-
 In Neo4j, dumps can be stored in the directory specified by the xref:configuration/configuration-settings.adoc#config_server.directories.dumps.root[`server.directories.dumps.root`] setting (by default, the path for storing dumps is xref:configuration/file-locations.adoc#data[`<neo4j-home>/data/dumps`]).
+Dumps can be used to create databases using the xref:clustering/databases.adoc#cluster-seed-uri[Seed from URI approach].
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:89]:
```suggestion
You can use dumps to create databases through the xref:clustering/databases.adoc#cluster-seed-uri[Seed from URI approach].
```

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 2]
```diff
@@ -85,12 +85,17 @@ By appending `DUMP DATA` to the command `DROP DATABASE`, you can create a dump o
...
 
 The option `DESTROY DATA` explicitly requests the default behavior of the command.
 
+[NOTE]
+====
+The dumps produced by `DUMP DATA` are equivalent to those produced by xref:backup-restore/offline-backup.adoc[`neo4j-admin database dump`]. 
+They can also be restored using the xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`] command.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:96]:
```suggestion
You can also restore them using the xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`] command.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2010.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-2010.surge.sh/operations-manual/2025.01/database-administration/standard-databases/delete-databases)

**Review by NataliaIvakina — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2010.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-2010.surge.sh/operations-manual/2025.01/database-administration/standard-databases/delete-databases)

**Comment by jackwaudby** [comment]
I've put the cherry pick to 5.x up https://github.com/neo4j/docs-operations/pull/2024

**Pull request merged**